rsconf/component/sirepo.py

```diff
@@ -185,14 +186,13 @@ def _install_dirs_and_files(self):
             self.install_directory(p.join(c))
         self.install_access(mode='400')
         for c in z.feature_config.proprietary_sim_types:
-            if c != 'jupyterhublogin':
```

Comment:
I still think this isn't quite right. There should be some feature test or config that a proprietary_code has proprietary_file. I think this should be `if c == 'flash'` then there's a file a to copy. It doesn't have to be a config for now, because we only have such code.

https://github.com/radiasoft/sirepo/issues/3376 discusses the general problem.

Reply:
https://github.com/radiasoft/rsconf/pull/141/commits/5469508058d2d9b78eae75e580d6b2913c09bcc6